makefiles/asdf.mk

```diff
@@ -1,5 +1,3 @@
-check-program = $(foreach exec,$(1),$(if $(shell PATH="$(PATH)" which $(exec)),,$(error "Missing deps: no '$(exec)' in PATH")))
-
 asdf.install_plugins:
 	$(call check-program, asdf)
```

Comment:
Is `check-program` found in the parent Makefile? Is it working like it?

Reply:
<img width="490" alt="Screenshot 2023-08-11 at 08 35 39" src="https://github.com/berty/weshnet-expo/assets/689440/183bd516-731e-4b26-8983-f46dc58f0ccd">

Yeah, that works. ;)